### DISCUSSION TITLE
Java JDK Version on Mac

### DISCUSSION THREAD

**Discussion by jo-ann1902** [irgroup-classrooms/dis17-2021#16] [Q&A]
Hello,

I cannot start Solr. If I run the command "./bin/solr start", it says: "Java not found, or an error was encountered when running java.
A working Java 8 JRE is required to run Solr!
Please install latest version of Java 8 or set JAVA_HOME properly.
Command that we tried: 'java -version', with response:
No Java runtime present, requesting install."
I tried to install JRE (wich worked) but I somehow cannot download a working JDK.
"java -version" prints "No Java runtime present, requesting install.".

Can someone help me with installing Java JDK?

**Comment by MehmetCagferoglu** [comment]
Hello,

here is a link that might help you!
https://www.youtube.com/watch?v=pxi3iIy4F5A
https://java.com/en/download/manual.jsp

Good luck,

sorry I am part of the Windows family ;)
